### ISSUE TITLE
MBD tree must be saved [WIP]

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/razzo#152]

**Comment by richelbilderbeek** [comment]
I think Giovanni would like this one :+1:

**Comment by Giappo** [comment]
2 things

1) We need to provide a path for the tree. Ideally I think it should be the path of the simulations we are using, so:
lambda-mu-nu-q/seed/tree_file
A way to get this information would be to use this:

directory <- dirname(twinning_params$twin_tree_filename)
  tree_filename <- file.path(
    directory,
    razzo_params$misc_params$tree_filename
  )
  ape::write.tree(
    phy = phylogeny, file = tree_filename
  )

Do you think it might be a good idea?

2) What kind of name do we wanna use?
I think the best thing should be the one we used to use in the previous version of razzo, so something like: mbd.tree (we already have bd.tree, so it looks like it's in line with it)

**Comment by richelbilderbeek** [comment]
Agreed, follow the same folder structure (go ahead and tweak it if you feel so) and within a folder, name it `mbd.tree`, so it can sit next to its friends like `bd.tree`, `mbd.fasta` and so :rainbow:

**Comment by richelbilderbeek** [comment]
Done!

![2019-06-17-104239_960x1056_scrot](https://user-images.githubusercontent.com/2098230/59590600-b8493d80-90ec-11e9-9da6-62bada2e5031.png)

**Comment by richelbilderbeek** [comment]
Done!

![2019-06-17-104239_960x1056_scrot](https://user-images.githubusercontent.com/2098230/59590600-b8493d80-90ec-11e9-9da6-62bada2e5031.png)
